packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -115,11 +118,23 @@ interface TooltipBaseProps extends VibeComponentProps {
    * Overwrites z-index of the tooltip
    */
   zIndex?: number;
-  // TODO: make defualt next major
+  // TODO: make default next major
   /**
    * Limit tooltip to 240px
    */
   withMaxWidth?: boolean;
+  /**
+   * The title of the tooltip
+   */
+  title?: string;
+  /**
+   * The image of the tooltip
+   */
+  image?: string;
```

Comment:
do we want to support only image? not `media`?
in `Tipseen` it pretty much messed up things when they initially did it for images only and I had to add a `TipseenMedia` component and had to manipulate to not make breaking changes

Reply:
We talked about it and decided to use just an image for now but I'll double check 👍🏼